server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/UploadRandomSeed.java

```diff
@@ -86,26 +83,16 @@ public String endpoint(final Request the_request, final Response the_response) {
           }
         } else if (dosd == null) {
           Main.LOGGER.error("could not get department of state dashboard");
-          status = HttpStatus.INTERNAL_SERVER_ERROR_500;
-          result = "Could not set random seed";
-        } else {
-          Main.LOGGER.info("attempt to set the random seed " +
-                           "in incorrect state " + dosd.auditStage());
-          status = HttpStatus.FORBIDDEN_403;
-          result = "Attempt to set the random seed in incorrect state";
+          serverError(the_response, "Could not set random seed");
         }
       } else {
         Main.LOGGER.info("attempt to set an invalid random seed");
-        status = HttpStatus.BAD_REQUEST_400;
-        result = "Invalid random seed specified";
+        invariantViolation(the_response, "Invalid random seed specified");
       }
     } else {
       Main.LOGGER.info("unauthorized attempt to set the random seed");
-      status = HttpStatus.UNAUTHORIZED_401;
-      result = "Unauthorized attempt to set the random seed"; 
+      unauthorized(the_response, "Unauthorized attempt to set the random seed"); 
```

Comment:
Yes, when we figure out how authentication will actually work.  You can see the commented out logic about it in the `before` method.